src/App.css

```diff
@@ -1,38 +1,97 @@
-.App {
-  text-align: center;
+/* red */
+
+/* :root {
+  --main-color: #e12d72;
+  --sec-color: #e9518b;
+  --third-color: #b93266;
+  --forth-color: #5788E4;
+} */
+
+/* blue */
+:root {
+  --main-color: #5788e4;
+  --sec-color: #4063a6;
+  --third-color: #4369b3;
+  --forth-color: #35548b;
+}
+
+.main-color {
+  background-color: var(--main-color);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.sec-color {
+  background-color: var(--sec-color);
+}
+
+.third-color {
+  background-color: var(--third-color);
+}
+
+.forth-color {
+  background-color: var(--third-color);
+}
+
+.item-card:nth-child(odd) {
+  /* background-color: #e12d72; */
```

Comment:
- [ ] kindly remove all commented code in this file for clean code practices

Reply:
done